tests/Unit/Validation/BankDataValidatorTest.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace WMDE\Fundraising\Frontend\Tests\Unit\Validation;
+
+use WMDE\Fundraising\Frontend\Domain\BankData;
+use WMDE\Fundraising\Frontend\Domain\Iban;
+use WMDE\Fundraising\Frontend\Validation\BankDataValidator;
+use WMDE\Fundraising\Frontend\Validation\IbanValidator;
+use WMDE\Fundraising\Frontend\Validation\ValidationResult;
+
+/**
+ * @covers WMDE\Fundraising\Frontend\Validation\BankDataValidator
+ *
+ * @license GNU GPL v2+
+ * @author Kai Nissen < kai.nissen@wikimedia.de >
+ */
+class BankDataValidatorTest extends ValidatorTestCase {
+
+	/**
+	 * @dataProvider invalidBankDataProvider
+	 * @param string $iban
+	 * @param string $bic
+	 * @param string $bankName
```

Comment:
Please omit param docs for test methods with data providers. I know that recent PHPStorm bitches about this (it used to not do that, I already filed an issue about it). The docs are out of date here btw :)